projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-following.php

```diff
@@ -57,6 +57,27 @@ public function register_routes() {
 				),
 			)
 		);
+
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/recommendations',
+			array(
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_recommendations' ),
+					'permission_callback' => array( $this, 'permission_check' ),
+					'args'                => array(
+						'number' => array(
+							'type'              => 'number',
+							'default'           => 5,
+							'validate_callback' => function ( $param ) {
+								return is_numeric( $param ) && $param <= 20;
+							},
```

Comment:
@heavyweight The REST API's internal validation engine handles this out of the box if you pass a `'maximum' => 20` argument instead. [When the type is `number`](https://github.com/WordPress/wordpress-develop/blob/6.3/src/wp-includes/rest-api.php#L2179-L2180) it [runs it through `is_numeric()`](https://github.com/WordPress/wordpress-develop/blob/6.3/src/wp-includes/rest-api.php#L2488) and checks for any floors and ceilings ([like the maximum](https://github.com/WordPress/wordpress-develop/blob/6.3/src/wp-includes/rest-api.php#L2523))